### PR TITLE
LLM Passthrough

### PR DESCRIPTION
**Pull request by amiller27**

### CODE REVIEW

**Comment by amiller27** [comment]
Tested in the sim

File: src/iarc7_motion/motion_planner.py [hunk 1]
```diff
@@ -48,7 +54,9 @@ def __init__(self, _action_server, update_rate):
             task_commands.VelocityCommand: self._handle_velocity_command,
             task_commands.ArmCommand: self._handle_arm_command,
             task_commands.NopCommand: self._handle_nop_command,
-            task_commands.GroundInteractionCommand: self._handle_ground_interaction_command
+            task_commands.GroundInteractionCommand: self._handle_ground_interaction_command,
+            task_commands.PassthroughMode: self._handle_passthrough_mode_command,
+            task_commands.PassthroughCommand: self._handle_passthrough_command
```
Comment (aftersomemath) [inline src/iarc7_motion/motion_planner.py]: AngleThrottleCommand? Really we should have names for the different controllers.

File: src/iarc7_motion/motion_planner.py [hunk 2]
```diff
@@ -48,7 +54,9 @@ def __init__(self, _action_server, update_rate):
             task_commands.VelocityCommand: self._handle_velocity_command,
             task_commands.ArmCommand: self._handle_arm_command,
             task_commands.NopCommand: self._handle_nop_command,
-            task_commands.GroundInteractionCommand: self._handle_ground_interaction_command
+            task_commands.GroundInteractionCommand: self._handle_ground_interaction_command,
+            task_commands.PassthroughMode: self._handle_passthrough_mode_command,
```
Comment (aftersomemath) [inline src/iarc7_motion/motion_planner.py]: I would rather this be named SwitchControllerCommand

File: src/LowLevelMotionController.cpp [hunk 3]
```diff
@@ -258,6 +271,13 @@ int main(int argc, char **argv)
         {
             last_time = current_time;
 
+            ROS_INFO_STREAM("server active: " << server.isPreemptRequested());
+            if (server.isPreemptRequested() && motion_state == MotionState::PASSTHROUGH) {
+                ROS_WARN("Transitioning out of PASSTHROUGH");
+                motion_state = MotionState::VELOCITY_CONTROL;
```
Comment (aftersomemath) [inline src/LowLevelMotionController.cpp]: Need a quadController.prepareForTakeover(); here.

**Pull request merged**